.github/workflows/docker.yml

```diff
@@ -12,9 +18,29 @@ jobs:
       - uses: actions/checkout@master
         with:
           fetch-depth: 0
-      - name: Get current date
-        id: date
-        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Prepare
+        id: prep
+        run: |
```

Comment:
I have no idea how to test this. It would be good if someone else understands this and takes a good eye to it (or you run it in some test run)

Reply:
i also test on my own fork: https://hub.docker.com/r/marbar3778/simapp/tags?page=1. Still ironing out something